fast64_internal/utility.py

```diff
@@ -523,6 +525,118 @@ def deleteIfFound(filePath, stringValue):
 			fileData.write(stringData)
 		fileData.close()
 
+def yield_children(obj: bpy.types.Object):
+	yield obj
+	if obj.children:
+		for o in obj.children:
+			yield from yield_children(o)
+
+def iter_from_active():
+	active_obj = bpy.context.view_layer.objects.active # keep original obj reference
+	yield from yield_children(active_obj)
+	bpy.context.view_layer.objects.active = active_obj
+
+def store_original_mtx():
+	for obj in iter_from_active():
+		obj['original_mtx'] = obj.matrix_local
+
+def rotate_bounds(bounds, mtx: mathutils.Matrix):
+	return [
+		(mathutils.Vector(b[:]) @ mtx).to_tuple()
+		for b in bounds[:]
```

Comment:
```suggestion
		(mathutils.Vector(b) @ mtx).to_tuple()
		for b in bounds
```
I don't think you need to make copies, do you

also is there a reason for `mathutils.Vector(b) @ mtx` instead of the more common `mtx @ mathutils.Vector(b)`

Reply:
weird yeah i don't need to make copies, for some reason i thought i had to when testing things out 🤔 changing that now.

> also is there a reason for `mathutils.Vector(b) @ mtx` instead of the more common `mtx @ mathutils.Vector(b)`

yesss i want to rotate the vector from the matrix, the other way around is, translating the matrix from the vector? not sure exactly but the other way jacks up the bounds